resources/assets/components/InboxItem/index.js

```diff
@@ -11,6 +11,14 @@ class InboxItem extends React.Component {
     }
   }
 
+  calculateAge(date) {
```

Comment:
interesting, we don't store age on the user? Regardless, this might be a useful helper instead of living on this specific component.